sw/device/lib/dif/dif_pwrmgr.h

```diff
@@ -251,12 +253,20 @@ typedef enum dif_pwrmgr_config_result {
  * Power manager interrupts.
  */
 typedef enum dif_pwrmgr_irq {
+  /**
+   * \internal First power manager interrupt.
+   */
+  kDifPwrmgrIrqFirst,
   /**
    * The device woke up from low power state.
    *
    * Note: This interrupt is not triggered during power-on reset.
    */
-  kDifPwrmgrIrqWakeup,
+  kDifPwrmgrIrqWakeup = kDifPwrmgrIrqFirst,
```

Comment:
[Nit] I think it might be better if we assume that interrupts go from `0` through to `kDifPwrmgrIrqLast` rather than adding a `First` interrupt, which makes the enum definition a bit more cluttered for the first "real" enum constant.

This sidesteps the problem you're seeking to avoid, which is a reordering of the interrupt names.

Reply:
Sounds good, done.